packages/wallets/package.json

```diff
@@ -18,6 +18,7 @@
         "classnames": "^2.2.6",
         "embla-carousel-react": "^8.0.0-rc12",
         "formik": "^2.1.4",
+        "framer-motion": "^6.5.1",
```

Comment:
We are using this library across our deriv-app. With this library, we can easily use and customize animations. Please put the comments here if we should not add this library to our wallets package.

Reply:
given that unpacked size of this library is 2.08MB, I would personally avoid adding it, 

if we need to have library for the animations, probably its worth having discussion, choose one and stick to it (reasons I'm saying that, is that in other PRs, I've seen different animation library being added)

Reply:
replied [here](https://github.com/binary-com/deriv-app/pull/10793#discussion_r1367808316)